src/views/Home.jsx

```diff
@@ -26,60 +31,109 @@ export function Home({ handleListTokenState }) {
 		if (placeholder) {
 			handleListTokenState(newToken);
 		} else {
-			setMessage('Error adding empty list to Firebase');
+			toast({
+				title: 'Error',
+				description: 'Error adding empty list to Firebase',
+				variant: 'errorToast',
+			});
 		}
 	};
 
 	const handleChange = (e) => setUserEnteredToken(e.target.value);
 	const handleFormSubmit = (e) => {
 		e.preventDefault();
 		if (userEnteredToken.length === 0) {
-			return setMessage('Please enter a token.');
+			toast({
+				title: 'Error',
+				description: 'Please enter a token.',
+				variant: 'errorToast',
+			});
+			return;
 		}
 		streamListItems(userEnteredToken, (snapshot) => {
 			if (snapshot.empty) {
-				setMessage('List not found. Please try another token.');
+				toast({
+					title: 'Error',
+					description: 'List not found. Please try another token.',
+					variant: 'errorToast',
+				});
 			} else {
 				handleListTokenState(userEnteredToken);
 			}
 		});
 	};
 
 	return (
-		<div
-			className="Home"
-			style={{
-				display: 'flex',
-				flexDirection: 'column',
-				alignItems: 'center',
-				gap: '1rem',
-			}}
+		<Flex
+			direction="column"
+			h="100%"
+			align="center"
+			justify="space-around"
+			pt={4}
 		>
-			<p>
-				Hello from the home (<code>/</code>) page!
-			</p>
-			<button onClick={handleCreateList}>Create a new list</button>
-
-			<p>or</p>
-			<form
-				onSubmit={handleFormSubmit}
-				style={{
-					display: 'flex',
-					flexDirection: 'column',
-					alignItems: 'center',
-					gap: '1rem',
-				}}
-			>
-				<label htmlFor="join-list">Join an existing list</label>
-				<input
-					type="text"
-					id="join-list"
-					value={userEnteredToken}
-					onChange={handleChange}
-				/>
-				<button>Join</button>
-				{message && <p>{message}</p>}
-			</form>
-		</div>
+			<Box pb={4}>
+				<Button
+					bg="soon.500"
+					textColor="brand.500"
+					onClick={handleCreateList}
+					fontSize={{ base: 'md', md: 'xl' }}
+				>
+					Create a new list
+				</Button>
```

Comment:
![image](https://github.com/the-collab-lab/tcl-55-smart-shopping-list/assets/99099243/3242abcf-cb29-4d52-8d66-2885703e6d7b)
Button is completely gone on the desktop version, while image is too large.